### ISSUE TITLE
Multiple Labels are totally allowed for radio buttons and checkboxes and more

### ISSUE THREAD

**Issue by goodwitch** [dequelabs/axe-core#1893]
axe is throwing the following error on a client's web page: "form-field-multiple-labels"

The sourcecode is

<input name="radioChildrens" id="radioChildrens-yes" aria-labelledby="lblradioChildrens lblradioChildrens-yes" aria-describedby="e-member.radioChildrens" type="radio" value="true">

Axe gives this explaination:
To solve this violation, you need to:
Fix the following:
Multiple label elements is not widely supported in assistive technologies

I don't think this should be an outright failure.  For radio buttons and checkboxes, this is a possible solution for associated the radio value and the actual question for the radio buttons.

I think aria-labeledby can totally support two labels and it is logical in this situation. 

What screen reader (or AT) cannot support this structure?

**Comment by straker** [comment]
We did some [extensive testing awhile ago](https://github.com/dequelabs/axe-core/issues/689#issuecomment-490176712), and depending on how the labels are set up certain browser/screen reader combinations wouldn't respect the multiple labels. However, the rule should be coming up as Needs Review rather than as a violation as of [axe-core 3.4.0](https://github.com/dequelabs/axe-core/blob/develop/CHANGELOG.md#340-2019-10-18). If this is not the case please let us know.

**Comment by goodwitch** [comment]
Hi Steven,  this is showing up as a violation (not a needs review).  Here is the screenshot our co-worker Angela just took
![violation](https://user-images.githubusercontent.com/8203866/68774532-de6aff00-05f2-11ea-86be-df7ba996b289.png)

**Comment by goodwitch** [comment]
Steven, Do you know who I can ask about when axe-core 3.4.0 will be available in the Attest Extension?

**Comment by WilcoFiers** [comment]
@goodwitch Attest ext will be updated next Monday.

**Comment by goodwitch** [comment]
@straker I wonder if when y'all were doing testing...if you included the use case of concatenated labels (like for the purpose of a radio or checkbox having a label for that radio/checkbox AND having a "label" for the related question).  I just created a super tiny file to test with at http://glendathegood.com/a11y/multiple%20labels/multi-label.html  and then tested with VoiceOver (on both Chrome and Safari)...and VO read the concatenated label for this radio button just fine. 

In other words...multiple labels that conflict are likely to be a problem...but concatenated labels for radio and checkboxes...(and even other fields)...should be just fine.  See what I consider to be a perfectly valid concatenated label for a plain ole text field in Example 1 at https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute

**Comment by goodwitch** [comment]
I just confirmed that concatenated labels are spoken correctly by NVDA, VoiceOver and JAWS with all normal browser pairings.  So...I think it would be VERY HELPFUL if this axe-core rule did NOT trigger an "needs review issue" on concatenated labels....because there are valid and working in AT.

**Comment by straker** [comment]
labels created using multiple `aria-labelledby` should be fine. It's only an issue if you use multiple `<label>` elements in conjunction with multiple `aria-lablelledby`. I took a look at your test page and it passes the `form-field-multiple-labels` rule as there are no `<label>` elements.

**Comment by goodwitch** [comment]
Cool.  I figured out what the problem was.  I didn't have direct access to the page in question and was only seeing a small part of the code.  Once I got to see all of the code in context I realized that this axe-core rule was 100% accurate.  Thanks for helping me look deeper.  Sorry to disturb y'all.
